src/templates/smfcfg.yaml.j2

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: 2022-present Intel Corporation
+# SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
+# Copyright 2019 free5GC.org
+#
+# SPDX-License-Identifier: Apache-2.0
+
+info:
+  version: 1.0.0
+  description: SMF initial local configuration
+
+configuration:
+  smfDBName: {{ smf_database_name }}
+  enableDBStore: false
+  enableUPFAdapter: true
+  debugProfilePort: 5001
+  mongodb:
+    name: {{ default_database_name }}
+    url: {{ default_database_url }}
```

Comment:
`mongodb` section doesn't exist in the official [config file](https://github.com/free5gc/free5gc/blob/main/config/smfcfg.yaml). Is that intended?

Reply:
As far as I know, we're not charming free5gc, this uses sd-core, but it's not exactly the same. I guess there's some custom configuration done there, this is the [actual config file](https://github.com/omec-project/smf/blob/master/config/smfcfg.yaml) directly from the OMF Github repo for the SMF service. Let me know if that clarifies or if I'm missing something.

Reply:
Deploying sd-core from the helm charts following the [deployment guide](https://docs.sd-core.opennetworking.org/sdcore-1.3/deployment/deployment5G.html) also results in that config file written in the workload.

Reply:
KK